src/main/resources/rules/rulesDefault.json

```diff
@@ -63,5 +63,27 @@
         "from": "$.hrid"
       }
     ]
+  },
+  {
+    "tag": "655",
```

Comment:
can we keep the tags sorted please, it'll be easier to track which one's are supported just by looking at the  file